.stylelintrc.js

```diff
@@ -86,14 +111,21 @@ module.exports = {
             }
         ],
         'font-family-name-quotes': 'always-unless-keyword',
-        // Turn off rules 
+        // Turn off rules
         'selector-class-pattern': null,
+        'selector-id-pattern': null,
         'function-no-unknown': null,
         'property-no-vendor-prefix': null,
         'value-no-vendor-prefix': null,
         'font-family-no-missing-generic-family-keyword': null,
         'scss/dollar-variable-colon-space-after': null,
         'scss/no-global-function-names': null,
-        'value-keyword-case': null
+        'value-keyword-case': null,
+        'media-query-no-invalid': null,
+        'selector-id-pattern': null,
+        'keyframes-name-pattern': null,
+        'scss/dollar-variable-pattern': null,
+        'scss/at-extend-no-missing-placeholder': null,
+        'no-descending-specificity': null,
```

Comment:
`no-descending-specificity` was the single most common item failing linting, and respecting it retrospectively could result in unexpected changes in styling.

`scss/at-extend-no-missing-placeholder` means we can't use `@extend` to avoid duplication of code for class selectors - in some cases this would even mean duplicating bootstrap css which we definitely don't want to do. Best to just disable the rule.